src/frontends/tensorflow_common/src/op/bias_add.cpp

```diff
@@ -16,10 +17,23 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_bias_add_op(const NodeContext& node) {
-    default_op_checks(node, 2, {"BiasAdd"});
+    default_op_checks(node, 2, {"BiasAdd"}, true);
     auto value = node.get_input(0);
     auto bias = node.get_input(1);
 
+    auto complex_type_mark_value = as_type_ptr<ComplexTypeMark>(value.get_node_shared_ptr());
+    auto complex_type_mark_bias = as_type_ptr<ComplexTypeMark>(bias.get_node_shared_ptr());
+    auto complex_type_inputs = (complex_type_mark_value || complex_type_mark_bias) ? true : false;
+    // validations prior to processing
+    if (complex_type_inputs) {
+        FRONT_END_GENERAL_CHECK(complex_type_mark_value != nullptr && complex_type_mark_bias != nullptr,
+                        "Mul gox complex and non-complex inputs. Inputs should be of same type.");
```

Comment:
```suggestion
    auto complex_type_inputs = (complex_type_mark_value && complex_type_mark_bias) ? true : false;
    // validations prior to processing
    if (complex_type_inputs) {
```
Let us assume that the graph is valid and consistent provided by TF. So no need to check